pkgs/development/python-modules/confusable-homoglyphs/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  unittestCheckHook,
+  wheel,
+  click,
+}:
+
+let
+  kebabToSnake = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+in
+buildPythonPackage rec {
+  pname = "confusable-homoglyphs";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = kebabToSnake pname;
+    hash = "sha256-uZUAHJsuG0zqDPXzhAp8eRiKjLutBT1pNXK9jBwexGA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
```

```suggestion
  build-system = [ setuptools ];
```
`wheel` is not needed for build-system.

---

ty!